### ISSUE TITLE
When square_resize_div_64 = False is set, an error occurs.

### ISSUE THREAD

**Issue by zhaoshuo1223** [roboflow/rf-detr#983]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

一、Problem
When processing downstream tasks, I want to try not changing the aspect ratio of the input image, so I set square_resize_div_64 = False.
However, the following error occurs.

<img width="881" height="87" alt="Image" src="https://github.com/user-attachments/assets/613c2c26-6565-498b-8836-f8641fae19c0" />

While examining the implementation of the function _build_train_resize_config in coco.py, I found that this parameter ultimately affects the parameter shown in the figure: "LongestMaxSize".

<img width="596" height="451" alt="Image" src="https://github.com/user-attachments/assets/5fff76c5-f95d-4d27-8b27-6552bc24b4ce" />

After modifying this parameter to 2000, the error changed to:

<img width="774" height="72" alt="Image" src="https://github.com/user-attachments/assets/c024bb37-8b78-4c29-bb85-b3adaadbcbef" />

I set multi_scale = False, so the max size of the shorter side should be 896, but the actual error shows that the shorter side exceeds 896.

### Environment

RF-DETR: Latest

### Minimal Reproducible Example

 model = RFDETRMedium(
        num_windows=2,      
        patch_size=16,     
        resolution=896,     
        num_classes=3,
        positional_encoding_size = 896//16,
    )

    model.train(
        # dataset_file = 'coco',
        #dataset_dir = r'D:\aotto\quexian-jiance\data-20260331\coco', 
         multi_scale = False,
        square_resize_div_64 = False
    )

### Are you willing to submit a PR?

- [x] Yes, I'd like to help by submitting a PR!

**Comment by Borda** [comment]
Could you pls share what version of RFDETR you are using?

**Comment by zhaoshuo1223** [comment]
> Could you pls share what version of RFDETR you are using?

I directly pulled the latest source code.

**Comment by zhaoshuo1223** [comment]
Should the preprocessing of data during validation and export remain consistent with that used during training? For example, if padding is used
